specs/agents/universal-profiling-integration.md

```diff
@@ -188,6 +203,6 @@ stack-trace-id        | uint8[16]
 count                 | uint16
 
 * *trace-id*: The APM W3C trace id of the trace which was active for the given profiling samples
-* *trace-id*: The APM W3C transaction id of the transaction which was active for the given profiling samples
+* *transaction-id*: The APM W3C transaction id of the transaction which was active for the given profiling samples
```

Comment:
Does the W3C spec contains anything about the "transactions" ? From what I recall it's only about `tracestate` and `traceparent` (with a `trace-id` and `parent-id` as fields) ([ref](https://www.w3.org/TR/trace-context/#trace-context-http-headers-format)) but I might definitely have missed something.

Reply:
Ooops, this should be `The APM W3C span id of the transaction`